packages/adapter-vercel/index.js

```diff
@@ -39,6 +39,32 @@ const plugin = function (defaults = {}) {
 			builder.rimraf(dir);
 			builder.rimraf(tmp);
 
+			if (fs.existsSync('./vercel.json')) {
+				const vercel_file = fs.readFileSync('./vercel.json', 'utf-8');
+				const vercel_config = JSON.parse(vercel_file);
+				const crons = /** @type {Array<unknown>} */ (
+					Array.isArray(vercel_config?.crons) ? vercel_config.crons : []
+				);
+				const GET_routes = builder.routes.filter((route) => route.methods.includes('GET'));
+
+				crons.forEach((cron) => {
+					if (!(typeof cron === 'object' && cron !== null && 'path' in cron)) {
+						return;
+					}
```

Comment:
the negation is slightly confusing (to me at least) and we don't need to check for a `path` property at this stage because it's handed in the next line

```suggestion
				for (cron of crons) {
					if (typeof cron !== 'object' || !cron) {
						continue;
					}
```

Reply:
Agreed that `for of` is the right way to go here, but on the contrary, `'path' in cron` is actually required -- the prior two conditions only narrow `cron` to type `object`. Trying to access `object.path` (even if it's with a question mark) results in a type error.

Reply:
ugh of _course_ we have to add runtime guards to uselessly appease the typechecker